patches/chromium/chore_modify_chromium_handling_of_mouse_events.patch

```diff
@@ -34,10 +38,29 @@ index 932351e288f37fd09ae1a43f44e8b51fb0caa4b8..4a0616bc210d234e51e564daabdd2ebd
    // Overridden from WidgetObserver.
    void OnWidgetThemeChanged(Widget* widget) override;
 diff --git a/ui/views/win/hwnd_message_handler.cc b/ui/views/win/hwnd_message_handler.cc
-index 2bd015be3178ab8dea012d6b1f71d13dd0548c14..759f00dd4e674d1dfca690b82e6e1820900ebf0c 100644
+index 2bd015be3178ab8dea012d6b1f71d13dd0548c14..1aab69e8df20c0c0fc29030ca45e15a64a1d0b74 100644
 --- a/ui/views/win/hwnd_message_handler.cc
 +++ b/ui/views/win/hwnd_message_handler.cc
-@@ -3144,15 +3144,19 @@ LRESULT HWNDMessageHandler::HandleMouseEventInternal(UINT message,
+@@ -3130,6 +3130,18 @@ LRESULT HWNDMessageHandler::HandleMouseEventInternal(UINT message,
+     is_pen_active_in_client_area_ = false;
+   }
+ 
++  LONG message_time = GetMessageTime();
++  CHROME_MSG msg = {hwnd(),
++                    message,
++                    w_param,
++                    l_param,
++                    static_cast<DWORD>(message_time),
++                    {CR_GET_X_LPARAM(l_param), CR_GET_Y_LPARAM(l_param)}};
++  ui::MouseEvent event(msg);
++
++  if (message == WM_NCRBUTTONDOWN)
++    delegate_->HandleMouseEvent(&event);
```

Comment:
Any reason this cannot be handled in `NativeWindowViews::PreHandleMsg` ?

Reply:
Yes - `PreHandleMsg` doesn't receive any `WM_NCRBUTTONDOWN`, or `WM_RBUTTONDOWN` in testing. It looks like they're being intercepted prior to `PreHandleMsg`.